<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>711</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="display"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>711</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="6">
        <widget class="QPushButton" name="Empty">
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <widget class="QPushButton" name="Print">
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
       <item row="9" column="7">
        <widget class="QPushButton" name="Clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QPushButton" name="Front">
         <property name="text">
          <string>Front</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="insertValue"/>
         </item>
         <item>
          <widget class="QComboBox" name="insertIndex"/>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="atIndex"/>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="PushBack">
         <property name="text">
          <string>PushBack</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QSpinBox" name="pushBackValue"/>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="At">
         <property name="text">
          <string>At</string>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QPushButton" name="PopBack">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>PopBack</string>
         </property>
        </widget>
       </item>
       <item row="7" column="7">
        <widget class="QPushButton" name="Size">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QPushButton" name="Capacity">
         <property name="text">
          <string>Capacity</string>
         </property>
        </widget>
       </item>
       <item row="6" column="7">
        <widget class="QPushButton" name="Back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>assign</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
